.travis.yml

```diff
@@ -18,7 +18,7 @@ before_script:
 script:
   - cargo fmt -- --check
```

Comment:
I think this line can be removed now because it's included in `scripts/clippy`? And in that case it's probably better to rename this script because it also does the formatting.

Alternatively, it can be removed from `scripts/clippy` :)

Reply:
Done, took the first suggestion!